test/Makefile

```diff
@@ -4,7 +4,7 @@ CMD_ECHO =
 # Important directories
 BUILD_DIR = ./Build
 WOLFHSM_DIR = $(CURDIR)/../
-WOLFSSL_DIR ?= $(CURDIR)/../../wolfssl
+WOLFSSL_DIR = $(CURDIR)/../../wolfssl
```

Comment:
Yup.  My bad.  I was resolving a compile issue where the include path was getting munged since wh_settings.h is now able to include wc_port.h (for XFENCE).  I'll revert it as this was not the problem.